internal/command/commands/operation.go

```diff
@@ -0,0 +1,123 @@
+package commands
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+
+	"github.com/dream11/odin/internal/backend"
+	"github.com/dream11/odin/pkg/table"
+)
+
+var operationClient backend.Operation
+
+type Operation command
+
+func (o *Operation) Run(args []string) int {
+	// Define flag set
+	flagSet := flag.NewFlagSet("flagSet", flag.ContinueOnError)
+	name:= flagSet.String("name", "", "name of the operation")
+	componentType:= flagSet.String("component-type", "", "component-type on which operations will be performed")
+
+	err := flagSet.Parse(args)
+	if err != nil {
+		o.Logger.Error("Unable to parse flags! " + err.Error())
+		return 1
+	}
+
+	if o.List {
+		emptyParameters := emptyParameters(map[string]string{"--component-type": *componentType})
+		if len(emptyParameters) == 0 {
+			o.Logger.Info("Listing all operation(s)")
```

Comment:
Can we print this after receiving validated response from backend

Reply:
I changed it but I think it will nice be if we print before receiving the response so that the user will know that he will be able to see the response in some time otherwise it will be a blank screen for few seconds and the user won't be knowing what is happening